Formula/re2.rb

```diff
@@ -29,10 +29,9 @@ class Re2 < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "abseil"
```

Comment:
relates to https://github.com/google/re2/commit/552c2b8f737f871b8bd00006e0f5d4036c5a5683